applications/crossbar/doc/user_authentication.md

```diff
@@ -12,9 +12,9 @@ Provides an auth-token via user credentials
 
 Key | Description | Type | Default | Required | Support Level
 --- | ----------- | ---- | ------- | -------- | -------------
-`account_name` | The account name of the user | `string(1..128)` |   | `false` |  
+`account_name` | The account name of the user | `string(1..128)` |   | `true` |  
```

Comment:
You are trying to change `support level` which is only a field for 2600Hz (and is populated based on the schema. Revert this change.
Also, `account_name` is not strictly required; `account_realm` or a `phone_number` assigned to the account serve the same purposes.